### PR TITLE
feat/add contribute button in home page

### PR DESCRIPTION
**Pull request by BlackStarkGoku**

### CODE REVIEW

File: src/components/ContributeBox.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Button } from "./ui/button";
+import Link from "next/link";
+import { GITHUB_REPO_URL } from "@/utils/constants";
+
+function ContributeBox() {
+  return (
+    <div className="flex flex-col items-center">
+      <h2 className="mb-10 font-medium text-lg md:text-xl">
+        Have ideas to make cairovm.codes better?
```
Comment (melnikga) [inline src/components/ContributeBox.tsx:10]: Change the text above the button to "Have ideas to make Starkflare better?"

File: src/components/ContributeBox.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      </h2>
+      <Link href={GITHUB_REPO_URL}>
+        <Button size={"padded"} variant={"contribute"}>
```
Comment (melnikga) [inline src/components/ContributeBox.tsx:13]: Return the default variant and size to the button

File: src/components/ui/button.tsx
Comment (melnikga) [inline src/components/ui/button.tsx:1]: Remove the changes from button.tsx

**Review by melnikga — CHANGES_REQUESTED** [review]
Hi, please make the following changes:
-Remove the changes from button.tsx
-Return the default variant and size to the button
-Change the text above the button to "Have ideas to make Starkflare better?"

**Review by melnikga — APPROVED** [review]
LGTM!

**Review by barabanovro — APPROVED** [review]

**Pull request merged**